config/generic-mellow-fly-e3-v2.cfg

```diff
@@ -0,0 +1,244 @@
+# Copyright (C) 2024 Lieven Vanhercke <lieven.vanh@gmail.com>
+
+# This file may be distributed under the terms of the GNU GPLv3 license.
```

Comment:
Please dont add your own copyright or details about licensing, this is covered by your commit, this is covered in viii in point 7 in https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md#guidelines
James